helmfile.d/charts/openstack-monitoring/values.yaml

```diff
@@ -3,3 +3,7 @@
 # Declare variables to be passed into your templates.
 openstackMonitoring:
   enabled: false
+serviceMonitor:
+  labels:
+    k8s-app: openstack-cloud-controller-manager
+  namespace: ~
```

Comment:
question: Why is this `~`, wouldn't it be better with `""`?

Reply:
yes, I fixed it